Swiftfin/Views/ItemEditorView/ItemEditorView.swift

```diff
@@ -0,0 +1,77 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Factory
+import JellyfinAPI
+import SwiftUI
+
+struct ItemEditorView: View {
+
+    @Injected(\.currentUserSession)
+    private var userSession
+
+    @EnvironmentObject
+    private var router: ItemEditorCoordinator.Router
+
+    @State
+    var item: BaseItemDto
+
+    // MARK: - Body
+
+    var body: some View {
+        contentView
+            .navigationBarTitle(L10n.metadata)
+            .navigationBarTitleDisplayMode(.inline)
+            .navigationBarCloseButton {
+                router.dismissCoordinator()
+            }
+            .onNotification(.itemMetadataDidChange) { notification in
+                guard let newItem = notification.object as? BaseItemDto else { return }
+                item = newItem
+            }
+    }
+
+    // MARK: - Content View
+
+    private var contentView: some View {
+        List {
+            ListTitleSection(
+                item.name ?? L10n.unknown,
+                description: item.path
+            )
+
+            Section {
+                RefreshMetadataButton(item: item)
+                    .environment(\.isEnabled, userSession?.user.isAdministrator ?? false)
+            } footer: {
+                LearnMoreButton(L10n.metadata) {
+                    TextPair(
+                        title: L10n.refresh,
+                        subtitle: "Default refresh with the ability to override metadata without affecting existing images."
```

Comment:
I changed the strings a little bit and I was a bit confused for what this option actually does. So, I took a look at the flags being sent to the server and tried to track down what each flag does, mainly `.default` but was unable to really understand what that flag specifically does. `fullRefresh` and `none` are more straightforward.

On the server, this will eventually be called to refresh the metadata for an item:

- https://github.com/jellyfin/jellyfin/blob/5e8c0fe40c0eb4c29b0f11cdf397c9880708e74b/MediaBrowser.Providers/Manager/MetadataService.cs#L74

Do you know what `.default` actually does?

Reply:
Here is where this is used on Web: https://github.com/jellyfin/jellyfin-web/blob/2b86995fc77d52b8682ed87fb0cf3b0f88d15553/src/components/refreshdialog/refreshdialog.js#L81

My understanding is that:

*Full:* Replace Metadata
*Default:* Get Missing / Update new or updated 
*None:* Do not change Metadata

Web has a checkmark for "Replace Images" but I went the route of having:

Refresh: Default, just get metadate for new/update files
Refresh Metadata: Replace Metadata but not images
Replace Images: Replace Images but not Metadata
Replace All: Get new items for all sources

From the web link I sent, it looks like 10.10 has some additional items we may want to account for like TrickPlay. Finally, when I first set this up I did recursive unset which should default to false. Do we want to enable that? Or make that a toggle?

This is making me wonder if I should have broken this out into a refreshMetadataView to accommodate checkmarks or other variations?

Reply:
I've removed "Replace" since I wasn't finding the difference I was expecting. Default seems to have the same impact as Full when I tested it one by one. I also localized the surviving strings.